fuse_constraints/fuse_plugins.xml

```diff
@@ -0,0 +1,121 @@
+<library path="lib/libfuse_constraints">
+  <class type="fuse_constraints::AbsoluteAccelerationAngular2DStampedConstraint" base_class_type="fuse_core::Constraint">
+    <description>
+    A constraint that represents either prior information about a 2D angular acceleration, or a direct measurement of
+    the 2D angular acceleration.
+    </description>
+  </class>
+  <class type="fuse_constraints::AbsoluteAccelerationLinear2DStampedConstraint" base_class_type="fuse_core::Constraint">
+    <description>
+    A constraint that represents either prior information about a 2D linear acceleration, or a direct measurement of
+    the 2D linear acceleration.
+    </description>
+  </class>
+  <class type="fuse_constraints::AbsoluteOrientation2DStampedConstraint" base_class_type="fuse_core::Constraint">
+    <description>
+    A constraint that represents either prior information about a 2D orientation, or a direct measurement of the
+    2D orientation.
+    </description>
+  </class>
+  <class type="fuse_constraints::AbsolutePosition2DStampedConstraint" base_class_type="fuse_core::Constraint">
+    <description>
+    A constraint that represents either prior information about a 2D position, or a direct measurement of the
+    2D position.
+    </description>
+  </class>
+  <class type="fuse_constraints::AbsolutePosition3DStampedConstraint" base_class_type="fuse_core::Constraint">
+    <description>
+    A constraint that represents either prior information about a 3D position, or a direct measurement of the
+    3D position.
+    </description>
+  </class>
+  <class type="fuse_constraints::AbsoluteVelocityAngular2DStampedConstraint" base_class_type="fuse_core::Constraint">
+    <description>
+    A constraint that represents either prior information about a 2D angular velocity, or a direct measurement of
+    the 2D angular velocity.
+    </description>
+  </class>
+  <class type="fuse_constraints::AbsoluteVelocityLinear2DStampedConstraint" base_class_type="fuse_core::Constraint">
+    <description>
+    A constraint that represents either prior information about a 2D linear velocity, or a direct measurement of
+    the 2D linear velocity.
+    </description>
+  </class>
+  <class type="fuse_constraints::AbsoluteOrientation3DStampedConstraint" base_class_type="fuse_core::Constraint">
+    <description>
+    A constraint that represents either prior information about a 3D orientation, or a direct measurement of the
+    3D orientation.
+    </description>
+  </class>
+  <class type="fuse_constraints::AbsoluteOrientation3DStampedEulerConstraint" base_class_type="fuse_core::Constraint">
+    <description>
+    A constraint that represents either prior information about a 3D orientation, or a direct measurement of the
```

Comment:
Should we clarify that the constraints are done in Euler space here? I realise the name kind of gives that away.